### PR TITLE
Autotest fix

### PR DESCRIPTION
**Pull request by khancyr**
Pep8 autotest file
Correct or remove unused variable
Move some function on common for reuse

### CODE REVIEW

File: Tools/autotest/apmrover2.py [hunk 1]
```diff
@@ -84,9 +84,13 @@ def init(self):
         if os.path.exists(buildlog):
             os.unlink(buildlog)
         try:
```
[the remark this answers is not in the document]
Comment (amilcarlucas) [reply Tools/autotest/apmrover2.py]: +1

[the remark this answers is not in the document]
Comment (khancyr) [reply Tools/autotest/apmrover2.py]: done

**Comment by khancyr** [comment]
rebase and correct

**Comment by peterbarker** [comment]
I've pulled in more stuff from this.  Please tidy the commits.

I don't think the user-takeoff changes are well-founded here.  Changing the state of the vehicle as we do in there isn't a good thing.  Perhaps calling it "guided_user_takeoff" and adding a chunky comment at the top if it would be sufficient to make it OK.

**Comment by peterbarker** [comment]
The takeoff-alt thing - have you seen it get stuck when taking off?

**Comment by rmackay9** [comment]
@peterbarker, @khancyr, I guess I'll leave you guys to decide what to do about this PR and Peter can merge when you're ready.

**Comment by peterbarker** [comment]
I've merged more of this.  Please rebase.

**Pull request closed without merging**